ale_linters/python/pyre.vim

```diff
@@ -0,0 +1,29 @@
+" Author: dsifford <dereksifford@gmail.com>
+" Description: A performant type-checker supporting LSP for Python 3 created by Facebook
+
+call ale#Set('python_pyre_executable', 'pyre')
+call ale#Set('python_pyre_use_global', get(g:, 'ale_use_global_executables', 0))
+
+function! ale_linters#python#pyre#GetExecutable(buffer) abort
+    return ale#python#FindExecutable(a:buffer, 'python_pyre', ['pyre'])
+endfunction
+
+function! ale_linters#python#pyre#GetCommand(buffer) abort
+    let l:executable = ale_linters#python#pyre#GetExecutable(a:buffer)
+
+    let l:exec_args = l:executable =~? 'pipenv$'
+    \   ? ' run pyre persistent'
+    \   : ' persistent'
+
+    return ale#Escape(l:executable) . l:exec_args
+endfunction
```

Comment:
Could you add some tests for these functions? Have a look at similar tests in the `test/command_callback` directory.

Reply:
Sure thing. I'll take a look more into this tomorrow afternoon(ish). I'll probably have to chime back here in this thread though at some point with questions since I've never actually used vader before (even though I've had it installed for like almost two years now :stuck_out_tongue_closed_eyes:)